src/Types.tsx

```diff
@@ -0,0 +1,39 @@
+export type ResponseSectionsResults = {
+    apiUrl: string;
+    id: string;
+    webTitle: string;
+    webUrl: string;
+}
+
+export type ResponseSearchResults = {
+    id: string;
+    type: string;
+    sectionId: string;
+    sectionName: string;
+    webPublicationDate: string;
+    webTitle: string;
+    webUrl: string;
+    apiUrl: string;
+    isHosted: boolean;
+    pillarId: string;
+    pillarName: string;
+}
+
+export type ResponseSearch = {
```

Comment:
You can name those types better if you will move `Response` to the end so for example `ResponseSearch` -> `SearchResponse`